front/pages/poke/[wId]/data_sources/[name]/index.tsx

```diff
@@ -137,22 +138,31 @@ const DataSourcePage = ({
         <div className="px-8 py-8"></div>
         <Page.Vertical align="stretch">
           <Page.SectionHeader title={`${dataSource.name}`} />
+          <div className="text-sm font-bold text-action-500">
+            <Link href={`/poke/${owner.sId}`}>&laquo; workspace</Link>
+          </div>
+
+          {dataSource.connectorId && (
+            <div className="flex flex-col text-sm text-action-500">
+              <Link
+                href={`https://cloud.temporal.io/namespaces/dust-prod.gmnlm/workflows?query=connectorId%3D%22${dataSource.connectorId}%22`}
```

Comment:
Since the namespace is an environment variable, should we return it from the `getServerSideProps` ?

Reply:
Yeah that would be nice will attempt to fix.

Reply:
(done-ish)